syntax/gitmessengerpopup.vim

```diff
@@ -10,6 +10,8 @@ syn match gitmessengerEmail '\%(\_^ \<\%(Author\|Committer\): \+.*\)\@<=<.\+>' d
 " Diff included in popup
 syn match diffRemoved "^ -.*" display
 syn match diffAdded "^ +.*" display
+syn match diffWordsRemoved "\[\-.\{-}\-\]" display
+syn match diffWordsAdded "{+.\{-}+}" display
```

Comment:
```suggestion
syn region diffWordsRemoved start=/\[\-/ end=/\-\]/
syn region diffWordsAdded start=/{+/ end=/+}/
```

Please use `syn region` for highlighting regions.

Reply:
Ah yes, good point.